git_machete/cmd.py

```diff
@@ -1678,12 +1695,25 @@ def usage(c=None):
             * displays the output of 'machete-status-branch' hook (see help on 'hooks'), if present;
             * optionally lists commits introduced on each branch if '-l'/'--list-commits' is supplied.
 
+            Using colors can be disabled with a '--color' flag set to 'never'. With '--color=always' git machete always emits colors and with '--color=auto' it emits colors only when standard output
+            is connected to a terminal. '--color=auto' is the default. When colors are disabled, relation between branches is representend in the following way:
+            |
+            o-<branch1> # green
+            |
+            | |
+            | x-<branch2> # red
+            |   |
+            |   ?-<branch3> # yellow 
+            |
+            m-<branch4> # dim
+
             Note: in practice, both yellow and red edges suggest that the downstream branch should be updated against its upstream.
             Yellow typically indicates that there are/were commits from some other branches on the path between upstream and downstream and that a closer look at the log of the downstream branch might be necessary.
             Grey edge suggests that the downstream branch can be slid out.
 
             Options:
             -l, --list-commits            Additionally lists the messages of commits introduced on each branch.
+            --color[=WHEN]                Colorize the output; WHEN can be 'always', 'auto', or 'never'. 
```

Comment:
`[=WHEN]` suggests the arg is optional... but the code below indicates it's actually required